### PR TITLE
#170 documentation for working with tags

### PR DESCRIPTION
**Pull request by BatJan**
As instructed by @dampee in this issue https://github.com/umbraco/UmbracoDocs/issues/895#issuecomment-400261030 I hereby open a PR for documenting the "Tags" property editor - This PR relates to issues #170 and #230 

This is the initial commit and it's work in progress and this should not currently be merged.

### CODE REVIEW

**Comment by BatJan** [comment]
@dampee Before I do the documentation for tags pre v7.6 would you please have a look at what I have added for 7.6+? Just so I know if it's ok before moving on. Might as well catch any issues or change requests up front even though the version pre 7.6 will be a tad different :-)

File: Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Tags.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+The Tags property editor allows you to add multiple tags to a node.
+
+## Data Type Definition Example
+
+![Data Type Definition Example](images/tags/configuration.png)
+
+Data can be saved in either CSV format or in JSON format. By default data is saved in CSV format. The difference between using CSV and JSON is that with JSON you can save a tag, which includes comma seperated values.
+
+Since the release of Umbraco 7.6 there are built-in property value converters, which means you don't need to worry about writing them yourself or parse the JSON output when choosing "JSON" in the storage type field. Therefore code example last on this page will work out of the box without further ado.
```
Comment (dampee) [inline Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Tags.md]: last sentence sounds a bit funny. Wat is "last"? Might be my limited knowledge of English.

**Comment by dampee** [comment]
Hi Jan,
You can review the (alpha) documentation about the versioning here: https://github.com/umbraco/UmbracoDocs/pull/907/
In case you want to add YAML already  :-)

Apart from that, we could already merge this in and do the multi version thing in another pull request.  Right?

Thanks for all the work!

**Comment by BatJan** [comment]
Hi Damiaan,
Great I'll have a look this evening. Yes totally forgot about the groups! :) 

I'd like to add the multi-version within the same PR if it's ok with you? Unless that delays things of course - Then I'll just make a new branch based on this one.

What branch is the #907 stuff on? Spotted a minor detail in the code example I would like to change. The date should be rendered using a combination of a p and time tag.

**Comment by dampee** [comment]
Hi Jan

Great, looking forward to your groups update! Thanks a lot!

To multi-version in one PR is fine for me.  But it seems that this will delay things because the documentation cannot handle the YAML yet. Worked on the PR yesterday a little bit.

The #907 stuff is on this branch: https://github.com/umbraco/UmbracoDocs/tree/temp-v8

**Comment by BatJan** [comment]
Added a PR for the versioned documentation at #910

**Comment by dampee** [comment]
Jan, this is awesome! You are awesome!

Thanks a lot for doing all the careful work.  You really nailed this one!

🖐5️⃣🦇🚀

**Comment by BatJan** [comment]
Thanks you Damiaan - Now you're making me blush 😃 And thank you for carefully testing and providing good feedback 🙏

**Comment by BatJan** [comment]
Oops! @dampee - Just noticed that there are some broken images here https://our.umbraco.com/documentation/Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Tags - Should I fix this in this branch? Or will I need to do a new PR? The issue is that I have added a folder in /images/tags/ called 7.6 but 7.6 is converted to 7_6 in the link...So I'll need to rename the folder to 7_6 of course :)

**Comment by sofietoft** [comment]
Hi @BatJan ,
I hope you are enjoying the summer 😎 

I think the easiest here would be to create a new PR with a fix for this folder name?

-Sofie

**Pull request merged**